cli/src/pixl_cli/main.py

```diff
@@ -103,11 +109,21 @@ def check_env(*, error: bool, sample_env_file: click.Path) -> None:
     default=None,
     help="Rate at which to process items from a queue (in items per second).",
 )
-@click.argument(
-    "parquet-path", required=True, type=click.Path(path_type=Path, exists=True, file_okay=True)
+@click.option(
+    "--num-retries",
+    "num_retries",
+    type=int,
+    show_default=True,
+    default=5,
+    help="Number of retries to attempt before giving up, 5 minute wait inbetween",
 )
-def populate(
-    parquet_path: Path, *, queues: str, start_processing: bool, rate: Optional[float]
+def populate(  # too many args
```

Comment:
Could we have `--num-retries` be part of `pixl start` instead? That would avoid the `too many args`.
And in any case, presumably it only make sense to retry processing if the processing is actually started? So `pixl populate --no-start ... --num-retries 5` doesn't make much sense but is a valid command currently.

Reply:
`pixl start` does a different thing though, it doesn't populate so you can't have the retries based off it. Could add a check that there's logical consistency between `--no-start` and `--num-retries`?

Reply:
Aah I see, in that case maybe just a log message/warning that `--num-retries` is ignored in case of `--no-start`

Reply:
done!